packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation_objc/QueueUtils.m

```diff
@@ -13,3 +13,7 @@ void FLTEnsureToRunOnMainQueue(dispatch_block_t block) {
     block();
   }
 }
+
+void FLTDispatchQueueSetSpecific(dispatch_queue_t queue, const void *key) {
+  dispatch_queue_set_specific(queue, key, (void *)key, NULL);
```

Comment:
revert this?

Reply:
I removed it from this PR although I don't think it was critical

Reply:
Do we still need at least 1 swift file for the swift target?

Reply:
I don't see the relation between this change and the Swift placeholder file. Before this was just moved from `QueueTestUtils` to `QueueUtils` because will be needed outside of tests (during the transition when `CameraPlugin` is in Swift and `FLTCam` is still in ObjC)